rnote-ui/src/canvas/canvaslayout.rs

```diff
@@ -87,6 +87,10 @@ mod imp {
                     (document.x + document.width) * total_zoom
                         + super::CanvasLayout::OVERSHOOT_HORIZONTAL,
                 ),
+                Layout::SemiInfinite => (
+                    (engine.document.x - Document::SHADOW_WIDTH) * total_zoom,
```

Comment:
lets use the recently added OVERSHOOT_HORIZONTAL const here

Reply:
Yes, I merged without checking too much on the changes...
Also OVERSHOOT_HORIZONTAL shouldn't be dependent on the zoom right?

Reply:
btw what is the difference between SHADOW_WIDTH and OVERSHOOT_HORIZONTAL?

Reply:
I wanted to separate them, so they can be tweaked independently :)